index.html

```diff
@@ -53,6 +53,218 @@ <h1>I'm Mariane <br/>
                 </div>
             </div>
         </section>
+        <section id="card-section">
+            <div class="card-container">
+                <div class="cards">
+                    <div>
+                        <img src="asset/snapshot1.svg" alt="screenshot" class="screenshot">
+                    </div>
+                    <div class="card-description">
+                        <div>
+                            <p class="card-title">Tonic</p>
+                            <div class="card-bg-info">
+                                <p>Canopy</p>
+                                <ul>
+                                    <li>Back End Dev</li>
+                                    <li>2015</li>
+                                </ul>
+                            </div>
+                        </div>
+                        <div class="card-text">
+                            <p>
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <div class="card-skills">
+                            <ul>
+                                <li>html</li>
+                                <li>css</li>
+                                <li>javascript</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div>
+                        <img src="asset/snapshot2.svg" alt="screenshot" class="screenshot">
+                    </div>
+                    <div class="card-description">
+                        <div>
+                            <p class="card-title">Multi-Post Stories</p>
```

Comment:
- [ ] Kindly make sure to wrap the title of the project into the heading `hx` tag. :wink: